extensions/guacamole-auth-jdbc/modules/guacamole-auth-jdbc-postgresql/src/main/java/org/apache/guacamole/auth/postgresql/conf/PostgreSQLGuacamoleProperties.java

```diff
@@ -302,5 +302,17 @@ private PostgreSQLGuacamoleProperties() {}
         public String getName() { return "postgresql-enforce-access-windows-for-active-sessions"; }
 
     };
+    
+    /**
+     * The maximum number of identifiers/parameters to be included in a single batch when
+     * executing SQL statements.
+     */
+    public static final IntegerGuacamoleProperty POSTGRES_BATCH_SIZE =
+            new IntegerGuacamoleProperty() {
+
+        @Override
+        public String getName() { return "postgres-batch-size"; }
+
+    };
```

Comment:
Please maintain established naming convention (`postgresql-*` for the property, `POSTGRESQL_*` for the constant).

Reply:
Updated to be more consistent with naming conventions